component/src/main/java/com/synopsys/integration/alert/component/diagnostic/model/AlertQueueInformation.java

```diff
@@ -0,0 +1,29 @@
+package com.synopsys.integration.alert.component.diagnostic.model;
```

Comment:
Don't forget to perform a clean and build to generate the copyright headers for the new classes.